1001-2000/LIT1509GadlaW.xml

```diff
@@ -102,21 +103,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   </div>
                   
                </div>
-               <div type="textpart" subtype="Taammer" xml:id="Taammer">
-                 
-                  <label>Taʾammǝr</label>
-                  <div type="textpart" subtype="chapter">
-                     <ab/>
-                  </div>
-               </div>
-               <div type="textpart" subtype="hymn" xml:id="hymn">
-                  <label>hymn</label>
-                  <div type="textpart" subtype="chapter">
-                     <ab/>
-                  </div>
-                  
```

Comment:
Are the relations to other entities missing? (Or am I just failing to find them?)

Perhaps:

```
<listRelation>
<relation name="dcterms:isPartOf" active="LIT1509GadlaW" passive="LIT7229DossierWP"></relation>
<relation name="ecrm:P129_is_about" active="LIT1509GadlaW" passive="PRS9848Walatta"/>
</listRelation>
```